### PR TITLE
Fixed markup in readme

### PR DESCRIPTION
**Pull request by a-stankevich**
Regular text was shown as code

### CODE REVIEW

**Comment by amine95bz** [comment]
hi 

can someone help me to use database in the repository?

**Comment by zodiac403** [comment]
Hi @a5hik ,
do the changes from @a-stankevich work? Would be great to have the readme fixed.

File: README.md [hunk 1]
```diff
@@ -68,28 +68,34 @@ The directives are structured like below.
 - CSS styling may be applied via the `as-sortable-placeholder` class
 - Additional classes may be applied via the `additionalPlaceholderClass` option provided to the as-sortable item. e.g.
     in JS:
-```$scope.sortableOptions = { additionalPlaceholderClass: 'some-class' };```
+```js
+$scope.sortableOptions = { additionalPlaceholderClass: 'some-class' };
+```
     in HTML:
```
Comment (zodiac403) [inline README.md]: In my opinion, this line should be plaint text.

Comment (a-stankevich) [inline README.md]: which one? it all looks good to me

**Pull request closed without merging**